src/Components/Comments/CommentList.jsx

```diff
@@ -0,0 +1,20 @@
+import CommentCard from "./CommentCard";
+
+
+export default function CommentList ({comments}) {
+
+return(
+    <>
```

Comment:
You're rendering a single element of a ul inside a fragment. Consider removing the fragment as you can render the ul just fine, and would be more semantic.